netconf_client/connect.py

```diff
@@ -15,6 +39,7 @@ def connect_ssh(
     key_filename=None,
     sock=None,
     timeout=120,
+    **ignore
```

Comment:
Ah sorry I missed this in my last review. What is this for?

Reply:
I added this because I see this is used frequently by * args, ** kwds to ignore redundant arguments, e.g ipv6 gets there